hello-world-apk-overrides/readme.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+This project is an identical clone of `hello-world-apk` module and designed to show you how to override the default values
```

Comment:
your call, i kind of like the cleanness of the base version

On Tue, Aug 23, 2016 at 10:29 PM, Chris Doyle <notifications@github.com>
wrote:

> In hello-world-apk-overrides/readme.md
> <https://github.com/chrisdoyle/gradle-fury/pull/26#discussion_r75985858>:
>
> > @@ -0,0 +1,38 @@
> > +# Purpose
> > +
> > +This project is an identical clone of `hello-world-apk` module and designed to show you how to override the default values
>
> I am thinking that we could simply apply the override modifications to the
> hello-world-apk project itself, which was intended to showcase this kind of
> configuration, instead of adding a second APK project.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chrisdoyle/gradle-fury/pull/26/files/b9691c75bb145423f38851b15729292c1f62bfb9#r75985858>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB4kGR2EObSMpQijW8_gXklUy96Xrcogks5qi6xvgaJpZM4Jqb_T>
> .
>